src/vivarium_ciff_sam/utilities.py

```diff
@@ -82,6 +82,13 @@ def delete_if_exists(*paths: Union[Path, List[Path]], confirm=False):
 #     return data
 
 
+def get_norm_from_quantiles(mean: float, lower: float, upper: float,
+                            quantiles: Tuple[float, float] = (0.025, 0.975)) -> stats.norm:
+    stdnorm_quantiles = stats.norm.ppf(quantiles)
+    sd = (upper - lower) / (stdnorm_quantiles[1] - stdnorm_quantiles[0])
```

Comment:
Is it ever possible for stdnorm_quantiles[1] == stdnorm_quantiles[0]? I assume not but maybe there's a corner case?

Reply:
Only if `upper == lower`, which is very much not a valid case. There are all sorts of invalid inputs that can be provided to this function that I'm not checking for, since it's only being used by me.